src/app/views/sessions/signin/signin.component.html

```diff
@@ -57,6 +57,7 @@
                       [(ngModel)]="signinData.otp"
                       #otp="ngModel"
                       required
+                      autocomplete="off"
```

Comment:
Chrome ignores `off` value for `autocomplete`. It might be better to set it to something else, `nein`, `rejected`, ... anything else would work, except `off`.

https://stackoverflow.com/questions/12374442

Reply:
We have seen that problem with some fields like username and password (which are more aggressivly hit by browsers with these auto complete things) but in our testing it does work for this field